### ISSUE TITLE
npm package requires ES6 processing

### ISSUE THREAD

**Issue by searler** [nsisodiya/react-json-viewer#5]
Babel via Webpack fails with the error

```
ERROR in ./~/react-json-viewer/index.js
Module parse failed: C:\work\react-pcap\node_modules\react-json-viewer\index.js Line 1: Unexpected reserved word
You may need an appropriate loader to handle this file type.
| import React, {Component} from "react";
| import ObjectInspector from "react-object-inspector";
|
```

The standard webpack.config.js explicitly excludes `node_modules`, and had to be modified
to apply Babel to the react-json-viewer package.

**Comment by nsisodiya** [comment]
@searler - I am trying to solve it ! I have added webpack.config.js too !
Current bundle.js is including react.js too !
How to exclude the react in bundle ?

**Comment by nsisodiya** [comment]
issue fixed from version 1.0.7 !

**Comment by searler** [comment]
Thank you!
